plugins/file/plugin.go

```diff
@@ -21,16 +26,78 @@ func init() {
 	plugins.RegisterPlugin("file", setupFile6, setupFile4)
 }
 
+//Record holds an IP lease record
+type Record struct {
+	IP        net.IP
+	leaseTime time.Duration
+	leased    int64
+}
+
 // StaticRecords holds a MAC -> IP address mapping
 var StaticRecords map[string]net.IP
 
 // DHCPv6Records and DHCPv4Records are mappings between MAC addresses in
 // form of a string, to network configurations.
 var (
 	DHCPv6Records map[string]net.IP
-	DHCPv4Records map[string]net.IP
+	DHCPv4Records map[string]Record
+	LeaseTime     time.Duration
+	filename      string
+	server        net.IPNet
```

Comment:
The variable name `server` is not descriptive in this case because you are storing a subnet. Can it be changed to `network` or something similar? Also, can it be a reference instead of the whole struct?

Reply:
Sure